app/changelog/buildStatsComponent.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
```

Comment:
all components should be inside the outer components folder.

I'd say this component is an organism, so place it inside `components/organisms/changelog/`